run_pretrained_openfold.py

```diff
@@ -230,18 +230,18 @@ def main(args):
     prediction_dir = os.path.join(args.output_dir, "predictions")
     os.makedirs(prediction_dir, exist_ok=True)
 
-    for fasta_file in os.listdir(args.fasta_dir):
+    for fasta_file in [f for f in os.listdir(args.fasta_dir) if os.path.splitext(f)[1] in [".fasta", ".fa"]]:
```

Comment:
Purely from a readability standpoint it might be worth extracting this list comprehension to a short function

Reply:
Works for me!